src/assets/js/data/helpers.js

```diff
@@ -11,6 +11,8 @@ export const promisify = (request) => {
       if (res.body) {
         return deferred.reject(new Error(res.body.error));
       }
+
+      console.log(err);
```

Comment:
Each of these errors should be handled separately by whoever is calling this promise. This might leak internal information, if the error message contains anything we don't want them to see.

Reply:
whoops left this in from when i was debugging